flexget/plugins/clients/qbittorrent.py

```diff
@@ -267,7 +267,7 @@ def add_entries(self, task, config):
             if maxdownspeed:
                 form_data['dlLimit'] = maxdownspeed * 1024
 
-            ratio_limit = entry.get('ratio_limit', config.get('ratio_limit'))
+            ratio_limit = str(entry.get('ratio_limit', config.get('ratio_limit')))
             if ratio_limit:
                 form_data['ratioLimit'] = ratio_limit
```

Comment:
AFAIK `'None'` is not falsy, wouldn't it be better to move the conversion below?

```suggestion
            ratio_limit = entry.get('ratio_limit', config.get('ratio_limit'))
            if ratio_limit:
                form_data['ratioLimit'] = str(ratio_limit)
```

Reply:
Your proposed modification is absolutely correct—I must admit that my previous approach was lacking in thorough consideration. Your deep understanding of Python truly stands out!